.github/workflows/integration_test.yml

```diff
@@ -24,29 +24,29 @@ jobs:
       - name: install libbpf
         uses: sustainable-computing-io/kepler-action@v0.0.3
         with:
-          ebpfprovider: bcc
+          ebpfprovider: libbpf
       # build kepler image
       - name: build and export Kepler image
         run: |
           make build_containerized
           make save-image
         env:
           IMAGE_REPO: "localhost:5001"
-          IMAGE_TAG: "devel"
           IMAGE_NAME: "kepler"
+          IMAGE_TAG: "devel"
           CTR_CMD: docker
           IMAGE_OUTPUT_PATH: ${{env.OUTPUT_DIR}}${{env.FILE_NAME}}
       # save kepler image
       - name: save Kepler image as artifact
         uses: actions/upload-artifact@v3
         with:
-          name: kepler
+          name: keplerlibbpf
```

Comment:
```suggestion
          name: kepler-libbpf
```

Reply:
these files didnt change in the PR , only got renamed

Reply:
Oh!! right!